internal/api/helpers.go

```diff
@@ -0,0 +1,78 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/google/uuid"
+)
+
+// AccessActorID is a custom type that represents
+// an API response where the value can be:
+// uuid.UUID - a single ID
+// "*" - a wildcard string, meaning "all".
+//
+// nolint:musttag // we have custom marshal/unmarshal logic for this type
+type AccessActorID struct {
+	ID  *uuid.UUID
+	All bool
+}
```

Comment:
tiny correction - this custom type should reflect either:
1. a _**single**_ uuid
2. a `"*"` wildcard string

on PUT calls, we'll pass a `[]AccessActorID` (or a list of ^)

```json
{
  "manage_actor_ids": [
    "*"
  ],
  "view_actor_ids": [
    "*"
  ],
  "manage_team_ids": [],
  "view_team_ids": []
}
```

on GET calls, we'll hydrate a list of `[]ObjectActorAccess`, each of which will have an `ID: AccessActorID` <- this is a single UUID/wildcard, since the encapsulating data model is what gets put into a list

```json
{
  "manage_actors": [
    {
      "id": "*",
      "name": "*",
      "email": "*",
      "type": "*"
    }
  ],
  "view_actors": [
    {
      "id": "*",
      "name": "*",
      "email": "*",
      "type": "*"
    }
  ]
}
```